docs/src/pages/blog/everything.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title:  Introducing the IBM FHIR Server - $everything operation
+description: Introducing the IBM FHIR Server - $everything operation 
+date:   2021-04-21
+---
+
+# Introducing the IBM FHIR Server - $everything operation
+In [IBM FHIR Server 4.7.0](https://github.com/IBM/FHIR/releases/tag/4.7.0), the IBM FHIR Server team turned on the `Patient/$everything` operation. In [IBM FHIR Server 4.6.1](https://github.com/IBM/FHIR/releases/tag/4.6.1), an external contributor and an IBMer, [Luis García](https://github.com/luisgarcc) implemented  the operation [#1044](https://github.com/IBM/FHIR/issues/1044). The core team merged the initial implementation of the `Patient/$everything` operation.
```

Comment:
I'd maybe flip this so that it goes chronological.  And in either case, I think could do without the last sentence here.  Just my 2 cents.

Reply:
changed slightly to reflect the real intent (the thanks and attribution)